app/customer.py

```diff
@@ -0,0 +1,64 @@
+import dataclasses
+import math
+
+from app.shop import Shop
+
+
+@dataclasses.dataclass
+class Customer:
+    name: str
+    location: list[int]
+    money: int
+    car: dict
+    product_cart: dict
+
+    @staticmethod
+    def fuel_to_shop(
+            fuel_cons: float, distance_to_shop: float, fuel_price: float
+    ) -> float:
+        return ((fuel_cons * distance_to_shop) / 100 * fuel_price) * 2
+
+    def dist_to_shop(self, shop_location: list) -> float:
+        return (
+            math.sqrt((shop_location[0] - self.location[0]) ** 2
+                      + (shop_location[1] - self.location[1]) ** 2)
+        )
+
+    def is_all_products_exist(self) -> bool:
+        return all(
+            product in Shop.products.keys()
+            for product in self.product_cart.keys()
+        )
```

Comment:
Change as follow and move this method to shop class
```suggestion
    def is_all_products_exist(self, customer_products: dict) -> bool:
        return all(
            customer_product in self.products.keys()
            for customer_product in customer_products.keys()
        )
```